Tests/KronosTests/ClockTests.swift

```diff
@@ -33,8 +33,8 @@ final class ClockTests: XCTestCase {
         let firstExpectation = self.expectation(description: "Clock sync calls first closure")
         let lastExpectation = self.expectation(description: "Clock sync calls last closure")
         Clock.sync(
-            first: { _ in lastExpectation.fulfill() },
-            completion: { _ in firstExpectation.fulfill() })
+            first: { _, _ in lastExpectation.fulfill() },
```

Comment:
can remove this change now

Reply:
no, unfortunately.

Reply:
![screen shot 2017-08-28 at 2 19 10 pm](https://user-images.githubusercontent.com/75067/29794221-e47e86fa-8bfb-11e7-8ab7-7c7078188c94.png)